build.sbt

```diff
@@ -1,14 +1,23 @@
 // This build is for this Giter8 template.
-// To test the template run `g8` or `g8Test` from the sbt session.
+// To test the template run `test`, `g8` or `g8Test` from the sbt session.
 // See http://www.foundweekends.org/giter8/testing.html#Using+the+Giter8Plugin for more details.
 lazy val root = (project in file("."))
   .enablePlugins(ScriptedPlugin)
   .settings(
     name := "quickstart-scala-sbt",
-    test in Test := {
-      val _ = (g8Test in Test).toTask("").value
+    Test / test := {
+      val _ = (Test / g8Test).toTask("").value
     },
-    scriptedLaunchOpts ++= List("-Xms1024m", "-Xmx1024m", "-XX:ReservedCodeCacheSize=128m", "-XX:MaxPermSize=256m", "-Xss2m", "-Dfile.encoding=UTF-8"),
-    resolvers += Resolver.url("typesafe", url("https://repo.typesafe.com/typesafe/ivy-releases/"))(Resolver.ivyStylePatterns)
-
-)
+    scriptedLaunchOpts ++= List(
+      "-Xms1024m",
+      "-Xmx1024m",
+      "-XX:ReservedCodeCacheSize=128m",
+      "-XX:MaxPermSize=256m",
+      "-Xss2m",
+      "-Dfile.encoding=UTF-8"
+    ),
+    resolvers += Resolver.url("typesafe", url("https://repo.typesafe.com/typesafe/ivy-releases/"))(
+      Resolver.ivyStylePatterns
+    ),
```

Comment:
I'm just now realizing that I'm not sure what we're fetching from this resolver 🤔 you think it's still needed?

Reply:
CI agrees with you.